releases/1.8/beta/kubeflow/bundle.yaml

```diff
@@ -4,53 +4,60 @@ docs: https://discourse.charmhub.io/t/3749
 applications:
   admission-webhook:
     charm: admission-webhook
-    channel: latest/beta
+    channel: 1.8/beta
     trust: true
     scale: 1
     _github_repo_name: admission-webhook-operator
+    _github_repo_branch: 1.8/beta
```

Comment:
```suggestion
    _github_repo_branch: track/1.8
```

Reply:
I see this is the only app in the file with a `_github_repo_branch`.  Should this line either be removed, or added for all the others? iirc we use this extra field in our [scanning scripts](https://github.com/canonical/kubeflow-ci/blob/3c4f999df3de0578a035d41aae468453121c17b5/scripts/images/get-all-images.sh#L10) for our [scanning workflow](https://github.com/canonical/bundle-kubeflow/blob/413855663ed0275f42a64ecdd41c1ed602cc528d/.github/workflows/scan-images.yaml#L53)

Reply:
Resolving since repo branches were added